<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswdModify</class>
 <widget class="QDialog" name="PasswdModify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PasswdModify</string>
  </property>
  <widget class="QLabel" name="id_lab_passwd_modify">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>密码修改</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_lab_new_passwd">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_lab_passwd_confirm">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>密码确认：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit_new_passwd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit_passwd_confirm">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_sure">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_lab_tip">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
